packages/front-end/components/Settings/ApiKeysModal.tsx

```diff
@@ -63,6 +63,18 @@ const ApiKeysModal: FC<{
       submit={onSubmit}
       cta="Create"
     >
+      {!secret && projects.length > 0 && (
+        <SelectField
+          label="Project"
+          initialOption="All Projects"
+          value={form.watch("project")}
+          onChange={(v) => form.setValue("project", v)}
+          options={projects.map((p) => ({
+            label: p.name,
+            value: p.id,
+          }))}
+        />
+      )}
```

Comment:
We might also need similar scoping for webhooks (posting feature definitions to the SDK). Otherwise the SDK will take in a bunch of irrelevant / leaked information.